core/src/stories/Migration/Migration.stories.mdx

```diff
@@ -20,9 +18,18 @@ Previously we have not had a set naming convention for our events. This has led
 components where some events has had a `custom` prefix, and others have had other ways of indicating whether
 or not it is a custom event. We have now settled on a naming convention for all events.
 Both **external**, events we expect you as a user to interact with, and **internal**, events that are used within
-our components to update state/react to changes.
+our components to update state/react to changes. 
 
-In short:
+## Prefix change
```

Comment:
Nice! Could we add the same sort of examples but for component names?

Reply:
How do you mean?

Reply:
Or maybe you mean what Alex mentioned below?